archunit/src/test/java/com/tngtech/archunit/library/dependencies/RealLifeGraph.java

```diff
@@ -0,0 +1,99 @@
+package com.tngtech.archunit.library.dependencies;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Multimap;
+
+import static com.tngtech.archunit.library.dependencies.EdgeTest.newEdge;
+
+public class RealLifeGraph {
```

Comment:
Maybe the edge cases covered by this graph should be described.

Reply:
If I would know the exact edge case, then yes :wink: But unfortunately this graph is super complex and I don't know exactly why all our other graphs did not uncover that unblocking issue I've introduced. I just knew it would probably cost me a couple of hours to find out exactly the constellation causing the issue and I did not want to spend that time.
Or do you have an idea what the problem could be?

Reply:
I see no obvious reason why this should fail (but I don't know your erroneous refactoring either). I would propose leaving this as it is. Maybe you _could_ describe the error you introduced during refactoring, but this makes only sense, if this can be explained in two or three sentences.